src/styles/_catalog.scss

```diff
@@ -13,10 +12,23 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width:1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (max-width:767px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
```

Comment:
You can move your media under general class using scss nesting to avoid writing the `.catalog` class multiple times

Reply:
I done. Thank you.